assets/out

```diff
@@ -1,8 +1,5 @@
 #!/bin/bash
 
-set -e
```

Comment:
Same here.

Reply:
will fix, I'll push the `set +e` just before I run helm upgrade